### PR TITLE
feat(typescript/amazon-mq-rabbitmq-lambda) Add an example of Amazon MQ RabbitMQ

### PR DESCRIPTION
**Pull request by architec**
This PR adds an example of **Amazon MQ RabbitMQ Integration with Lambda, Secrets Manager, and CloudWatch Logs**. It demonstrates how to set up a RabbitMQ broker, a Lambda function to process messages, and secure credential management using AWS Secrets Manager. More details are provided in the README.

---

By submitting this pull request, I confirm that my contribution is made under the terms of the [Apache 2.0 license].

[Apache 2.0 license]: https://www.apache.org/licenses/LICENSE-2.0

### CODE REVIEW

**Review by kaiz-io — APPROVED** [review]
:100:

File: typescript/amazon-mq-rabbitmq-lambda/lib/amazon-mq-rabbitmq-lambda-stack.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    // Create a custom CloudWatch Log Group for the consumer Lambda function
+    const consumerLambdaLogGroup = new logs.LogGroup(this, 'ConsumerLambdaLogGroup', {
+      logGroupName: 'customLogGroup', // Custom log group name
+      removalPolicy: RemovalPolicy.DESTROY, // Ensure it's deleted on stack destroy
+    });
+
+    // Define the consumer Lambda function which will handle messages from RabbitMQ
+    const consumer_lambda = new Function(this, 'consumer_lambdaFunction', {
+      runtime: Runtime.NODEJS_20_X,       // Use Node.js 20.x runtime for the Lambda function
+      code: Code.fromAsset('lambda'),     // Path to Lambda function code directory
```
Comment (kaiz-io) [inline typescript/amazon-mq-rabbitmq-lambda/lib/amazon-mq-rabbitmq-lambda-stack.ts:54]: Looks like you forgot to add your lambda folder with the code.

**Review by kaiz-io — CHANGES_REQUESTED** [review]
Just need the lambda code and looks great.

File: typescript/amazon-mq-rabbitmq-lambda/lib/amazon-mq-rabbitmq-lambda-stack.ts (hunk 1 above)
Comment (architec) [reply typescript/amazon-mq-rabbitmq-lambda/lib/amazon-mq-rabbitmq-lambda-stack.ts:54]: I've added the missing lambda folder in a new commit. Thanks for pointing it out!

**Review by kaiz-io — APPROVED** [review]
:+1:

**Pull request merged**